### PR TITLE
fix: active link, navbar transparency

### PR DESCRIPTION
**Pull request by a-wong-8**
Fixed:
1. active link underline for navbar and footer
2. navbar transparency and footer overrun 

Screenshot 2024-08-13 at 10 31 50 AM
Screenshot 2024-08-13 at 10 31 45 AM

### CODE REVIEW

**Review by ridham-virufy — APPROVED** [review]
Great Work!

**Review by gevuong — CHANGES_REQUESTED** [review]
Is there a reason why the package lock changes are needed?

File: app/[lang]/our-story/page.tsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ const OurStoryPage = ({ params: { lang } }: { params: { lang: Locale } }) => {
   } = getDictionary(lang);
 
   return (
-    <div className="relative">
+    <div className="relative -top-24">
       {/* About Us/Hero Section */}
       <section>
-        <div className="relative bg-gradient-to-b from-black to-[#162b4c]">
+        <div className="relative bg-gradient-to-b from-black to-[#162b4c] ">
```
Comment (gevuong) [inline app/[lang]/our-story/page.tsx:19]: remove whitespace

**Comment by gevuong** [comment]
can you remove the package lock changes? I still see changes to that file

**Comment by a-wong-8** [comment]
> can you remove the package lock changes? I still see changes to that file

@gevuong Can you comment on the lines you want changed? I already tried deleting package json and running npm install. I dont know how to change it back

**Comment by gevuong** [comment]
@a-wong-8 I think there may be a bug. For example, when starting at the home page, when I select "Publications" in the top navbar, I don't see the underline under "Technology" in the footer. Similarly, if I select 'One Young World' (or any dropdown option under "About Us" except for "About Us") in the top navbar, I don't see the underline under "About Us" in the footer.

**Comment by a-wong-8** [comment]
> @a-wong-8 I think there may be a bug. For example, when starting at the home page, when I select "Publications" in the top navbar, I don't see the underline under "Technology" in the footer. Similarly, if I select 'One Young World' (or any dropdown option under "About Us" except for "About Us") in the top navbar, I don't see the underline under "About Us" in the footer.

@gevuong good catch

**Comment by a-wong-8** [comment]
@gevuong ready for review.

**Review by gevuong — APPROVED** [review]

**Pull request merged**